crates/web-sys/webidls/enabled/Range.webidl

```diff
@@ -10,10 +10,10 @@
  *
  * Copyright © 2012 W3C® (MIT, ERCIM, Keio), All Rights Reserved. W3C
  * liability, trademark and document use rules apply.
- */
+ */ 
 
 [Constructor]
-interface Range {
+interface Range : AbstractRange {
   [Throws]
   readonly attribute Node startContainer;
```

Comment:
(I'm not a reviewer and this isn't a review.) I've never worked with this before, so this might be a stupid question, but why is this attribute still defined here? 

The [IDL for `Range`](https://dom.spec.whatwg.org/#range) seems to rely on inheritance to get this attribute, so is the IDL parser wasm bindgen uses not smart enough to get inheritance or could this attribute (and the others defined by `AbstractRange`) be removed from this file?

Reply:
Thanks for the question! Here’s a bit more context:
- I added the inheritance from `AbstractRange` to `Range` in the IDL but left the existing attributes in place.
- I didn’t modify any bindings or attributes in `Range` itself because I wasn’t sure if removing the attributes would lead to breaking changes, especially with our current wasm-bindgen setup.
- The IDL parser that wasm-bindgen uses might not fully handle inheritance in this case, so removing the attribute definitions could potentially break the interface. 
That’s why I opted to keep the attributes defined in `Range` for now while adding the inheritance from `AbstractRange` to ensure compatibility.
Hope that clarifies things!